src/clj/puppetlabs/services/versioned_code_service/versioned_code_core.clj

```diff
@@ -16,25 +17,25 @@
   success-with-stderr-msg :- schema/Str
   [cmd :- schema/Str
    stderr :- schema/Str]
-  (format "Error output generated while running '%s'. stderr: '%s'"
-          cmd stderr))
+  (format "%s %s"
+          (trs "Error output generated while running ''{0}''." cmd)
+          (trs "stderr: ''{0}''" stderr)))
 
 (schema/defn ^:always-validate
   nonzero-msg :- schema/Str
   [cmd :- schema/Str
    exit-code :- schema/Int
    stdout :- schema/Str
    stderr :- schema/Str]
-  (format (str "Non-zero exit code returned while running '%s'. "
-               "exit-code: '%d', stdout: '%s', stderr: '%s'")
-          cmd exit-code stdout stderr))
+  (format "%s %s"
+          (trs "Non-zero exit code returned while running ''{0}''." cmd)
+          (trs "exit-code: ''{0}'', stdout: ''{1}'', stderr: ''{2}''" exit-code stdout stderr)))
 
 (schema/defn ^:always-validate
   execution-error-msg :- schema/Str
   [cmd :- schema/Str
    e :- Exception]
-  (format (str "Running script generated an error. "
-               "Command executed: '%s', error generated: '%s'")
+  (trs "Running script generated an error. Command executed: ''{0}'', error generated: ''{1}''"
```

Comment:
It's possible that these execution error messages might bubble back up through the HTTP API endpoints like `catalog`, etc.; might be worth auditing.